src/lib/core/exception.h

```diff
@@ -73,6 +73,7 @@ class Exception: public error {
 	int get_line() const { return line; }
 	const char *get_errmsg() const { return errmsg; }
 
+	virtual struct error *dup() const = 0;
```

Comment:
Please add a short comment to make checkpatch happy.

Reply:
Done.